build/deploy/db-schemas/defaultdb.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
+  data:{
```

Comment:
is there a way to import a whole directory?

Reply:
I looked and I can't find it